lib/matplotlib/ticker.py

```diff
@@ -675,7 +675,6 @@ def get_offset(self):
         """
         if len(self.locs) == 0:
             return ''
-        s = ''
         if self.orderOfMagnitude or self.offset:
```

Comment:
I would maybe swap this `if`, `return ''` first, and dedent everything here.

Reply:
This would give fewer line changes, and my experience is that changing code for this type of changes is not always appreciated. So I find it a bit confusing when it should be done or not (there are probably quite a lot of cases where similar changes can be done, one obvious being if-else clauses avoiding having a negated case first).